S3Scanner/__main__.py

```diff
@@ -181,13 +184,23 @@ def main():
         if args.dangerous:
             print("INFO: Including dangerous checks. WARNING: This may change bucket ACL destructively")
 
+        if args.json_file: # clear the results file
```

Comment:
Instead of clearing the file here, let's just always append to the file if it already exists and if it doesn't, create it then append.